static/.htaccess

```diff
@@ -1,16 +0,0 @@
-# Set far-future Expires headers for static media
-ExpiresActive on
-
-ExpiresDefault "access plus 1 week"
-ExpiresByType text/css "access plus 1 year"
-ExpiresByType text/javascript "access plus 1 year"
-ExpiresByType image/png "access plus 1 month"
-ExpiresByType image/jpeg "access plus 1 month"
-ExpiresByType image/gif "access plus 1 month"
-ExpiresByType image/vnd.microsoft.icon "access plus 1 month"
-ExpiresByType video/webm "access plus 1 week"
-ExpiresByType video/ogg "access plus 1 week"
-ExpiresByType video/x-flv "access plus 1 week"
-ExpiresByType application/x-shockwave-flash "access plus 1 week"
```

Comment:
Agreed. To my knowledge, the only pages referencing non-hashed assets are the pages of our Read the Docs theme. We can even bump the default value to "access + 1 week" since that's the lowest value we use now.